### DISCUSSION TITLE
12.0 Actions drop-down for "Menu Items" shows all the window actions

### DISCUSSION THREAD

**Discussion by raviprasadmr** [OCA/OpenUpgrade#3592] [Q&A]
Dear OCA,

I never really faced this issue before and suddenly from today I am seeing a weird behavior.

When I click on Settings > Menu Items > (Select any Menu) > Click on "Action" dropdown, I am supposed to only see 
Export, Archive, Unarchive, Delete options. However I see literally all the window actions ! How can I solve this?

**Comment by pedrobaeza** [comment]
I don't think this is related with OpenUpgrade at all.

**Reply by raviprasadmr** [reply]
It happened after migrating from OpenERP 7.0 to Odoo12 using OpenUpgrade. I am not blaming but just trying to get an opinion.

**Reply by pedrobaeza** [reply]
I have never faced this, sorry.
